src/Analyser.php

```diff
@@ -6,12 +6,11 @@
 
 namespace OpenApi;
 
-use Exception;
```

Comment:
Is removing  `use Exception` but adding a slash a common convention / better? (i genuinely don't know)

Reply:
As long as you don't forget the backslash, neither is actually better I believe. Leaving out the backslash does make it marginally slower because it would have to search both the current namespace and the global namespace...

It depends on the code-style really. Personally I always import (`use`) everything.

Reply:
It is a style/preference question. I am leaning towards the Symfony styleguide for the most part and that is how its used elsehwere in the codebase that I have touched.

Reply:
@bfanger  Reading through this again - when would you be ok with bumping this then? Active support for PHP 7.3 runs out next month and for PHP 7.4 in a year. After that, at the latest, 1.7 would be the min required version IMO...
In practice I would be surprised if this change would break things for anyone.

Reply:
What about  ">=1.7" ? Our usage of doctrine could be compatible with 2.x and up.

Probably a good idea to and bump "php": ">=7.4" and drop 7.3 and lower from travis.

Reply:
Back to the semver question, eh? I do not agree that "could be compatible" is good enough and who knows if/when that would be. From my experience, once there is a 2.0 we can test and if it is compatible bump deps then.
Noone expects a library to support a new major version of another library from day one, IMHO.
In fact, in that respect the current `"php": ">=7.2"` was wrong for quite a while because PHP8 was broken...

Reply:
It's a broken system, but better than no system at al. If we choose ^1.7 will Composer will select the current swaggr-php version? (as * is compatible with 2). But that aside going for a more safe settings seems like a good. path forward. Merge approved